roles/v2/test-utils/pool/src/main.rs

```diff
@@ -73,9 +69,81 @@ async fn server_pool() {
     }
 }
 
+mod args {
```

Comment:
if you choose to update this crate, move `mod arges` to `roles/v2/test-utils/pool/src/arg.rs` file